src/excmds.ts

```diff
@@ -218,7 +193,7 @@ export async function editor() {
 }
 
 /**
- * Behaves like readline's [delete_char](http://web.mit.edu/gnu/doc/html/rlman_1.html#SEC14).
+ * Behaves like readline's [delete_char](http://web.mit.edu/gnu/doc/html/rlman_1.html#SEC14), i.e. deletes the character to the right of the caret.
```

Comment:
I think it's clearer if you call it "the cursor". Carets are used for inserting text but it also means the literal `^` character, as per https://en.wikipedia.org/wiki/Caret, so it might be slightly confusing.